### PR TITLE
Move tolerance to param

### LINKED ISSUE #164: Simple Navigator
In simple navigator, endpoints tolerance is hand coded to a constant. This should be parameterized so that the user can change it.

### PR DESCRIPTION
**Pull request by mjeronimo**
Per the team discussion, move the tolerance value out of the PathEndPoints message to become a parameter of the global planners. Fixes #164.

### CODE REVIEW

**Comment by mjeronimo** [comment]
@SteveMacenski I agree with you and argued that the tolerance should be in the NavigateToPose message and would be something that would be task specific and not an overall robot parameter.  Others argued that it would be a parameter of the robot and not the task. Let's review this item at the WG meeting to get broader input. If the tolerance is associated with the task, it should be in the command to NavigateToPose and could therefore be modified on a per-task basis.

**Comment by SteveMacenski** [comment]
I think `NavigateToPose` is the exact right place for it. Lets tag up on it tomorrow then 

S

File: nav2_bringup/launch/system_test.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ComputePathToPoseNode:
+    ros__parameters:
```
[the remark this answers is not in the document]
Comment (mjeronimo) [reply nav2_bringup/launch/system_test.yaml]: @SteveMacenski  Yes, that's the format required by ROS2.

File: nav2_astar_planner/src/astar_planner.cpp [hunk 2]
```diff
@@ -28,6 +28,8 @@ AStarPlanner::AStarPlanner()
 : nav2_tasks::ComputePathToPoseTaskServer("ComputePathToPoseNode")
 {
   RCLCPP_INFO(get_logger(), "Initializing");
+
+  get_parameter_or_set("tolerance", tolerance_, 1.0);
```
[the remark this answers is not in the document]
Comment (mjeronimo) [reply nav2_astar_planner/src/astar_planner.cpp]: @SteveMacenski  Sure, I'll update.

**Pull request closed without merging**